core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -291,7 +291,6 @@ public StateMachineStorage getStateMachineStorage() {
 
   @Override
   public void close() {
-    mSnapshotManager.close();
```

Comment:
why is this gone?

Reply:
Got it. That's OK when all clients are created within try block. Otherwise it would stall server shutdown.